lib/datasource.js

```diff
@@ -799,16 +809,24 @@ DataSource.prototype.automigrate = function (models, cb) {
     });
 
     if (invalidModels.length) {
-      return process.nextTick(function () {
+      var error = process.nextTick(function () {
```

Comment:
AFAIK, `process.nextTick` does not return any error object, I think it returns `undefined`. Please simplify this block.